tests/run.sh

```diff
@@ -174,6 +174,9 @@ if [ -n "$DO_COVERAGE" ] && [ "$DO_COVERAGE" != 0 ]; then
     sed "1 s~^~require('luacov.runner')('$source_dir/.luacov'); \0~" \
         "$RC_FILE" > "$tmp_files/awesomerc.lua"
     RC_FILE=$tmp_files/awesomerc.lua
+
+    # Use xresources theme for more coverage.
+    sed -i 's~default/theme~xresources/theme~' "$RC_FILE"
```

Comment:
Doesn't this cause less coverage since now only the xresources theme is measured?
Also, if for some reason some error only appears under coverage, things might get confusing ("why does it break? it worked just a moment before without coverage?!?")